inference-engine/tests/functional/plugin/cpu/single_layer_tests/pad.cpp

```diff
@@ -109,95 +133,142 @@ const std::vector<CPUSpecificParams> CPUParams4DBlocked = {
+                ::testing::ValuesIn(padsBegin4D),
+                ::testing::ValuesIn(padsEnd4D),
+                ::testing::ValuesIn(argPadValue),
+                ::testing::Values(ngraph::helpers::PadMode::CONSTANT),
+                ::testing::Values(cpuParams_nhwc)),
+        PadLayerCPUTest::getTestCaseName
+);
 
-const auto pad4DConstParams = testing::Combine(
-        testing::ValuesIn(padsBegin4D),
-        testing::ValuesIn(padsEnd4D),
-        testing::ValuesIn(argPadValue),
-        testing::Values(ngraph::helpers::PadMode::CONSTANT),
-        testing::ValuesIn(inputPrecisions),
-        testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-        testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-        testing::Values(InferenceEngine::Layout::ANY),
-        testing::Values(std::vector<size_t>{3, 16, 5, 5}),
-        testing::Values(CommonTestUtils::DEVICE_CPU)
+INSTANTIATE_TEST_SUITE_P(
+        smoke_CPUPad4DBlocked,
+        PadLayerCPUTest,
+        ::testing::Combine(
+                ::testing::ValuesIn(static_shapes_to_test_representation({{3, 16, 10, 5}})),
+                ::testing::ValuesIn(inputPrecisions),
+                ::testing::ValuesIn(padsBegin4DBlocked),
+                ::testing::ValuesIn(padsEnd4DBlocked),
+                ::testing::Values(0),
+                ::testing::ValuesIn(padMode),
+                ::testing::ValuesIn(CPUParams4DBlocked)),
+        PadLayerCPUTest::getTestCaseName
 );
 
 INSTANTIATE_TEST_SUITE_P(
-        smoke_CPUPad4DConst,
+        smoke_CPUPad4D,
         PadLayerCPUTest,
         ::testing::Combine(
-                pad4DConstParams,
+                ::testing::ValuesIn(static_shapes_to_test_representation({{3, 16, 10, 5}})),
+                ::testing::ValuesIn(inputPrecisions),
+                ::testing::ValuesIn(padsBegin4DBlocked),
+                ::testing::ValuesIn(padsEnd4DBlocked),
+                ::testing::Values(0),
+                ::testing::ValuesIn(padMode),
                 ::testing::Values(cpuParams_nhwc)),
         PadLayerCPUTest::getTestCaseName
 );
 
-const auto pad4DParamsBlocked = testing::Combine(
-        testing::ValuesIn(padsBegin4DBlocked),
-        testing::ValuesIn(padsEnd4DBlocked),
-        testing::Values(0),
-        testing::ValuesIn(padMode),
-        testing::ValuesIn(inputPrecisions),
-        testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-        testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-        testing::Values(InferenceEngine::Layout::ANY),
-        testing::Values(std::vector<size_t>{3, 16, 10, 5}),
-        testing::Values(CommonTestUtils::DEVICE_CPU)
+/* *======================* *=====================* *======================* */
+
+/* *======================* Dynamic Shapes Tests 4D *======================* */
+
+const std::vector<InputShape> inputShapesDynamic4D = {
+        {{{-1, -1, -1, -1}},                                    // dynamic
```

Comment:
looks like to many braces around PartialShape: InputShape{ PartialShape{}, std::vector<Shape>{ Shape{}, Shape{}, Shape{} }  }. Here and for 5D